plugin/output/file/file.go

```diff
@@ -83,6 +86,26 @@ type Config struct {
 	//> File mode for log files
 	FileMode  cfg.Base8 `json:"file_mode" default:"0666" parse:"base8"` //*
 	FileMode_ int64
+
+	//> MetaCfg describes metafile with sealfile desription
+	MetaCfg MetaConfig `json:"meta_config" child:"true"`
+}
+
+type MetaConfig struct {
+	//> Stores metadata of sealed files.
+	StoreMeta bool `json:"store_meta"` //*
+	//> MetaDataFile saves metainformation about sealed file.
+	MetaDataDir string `json:"metadata_dir" default:"/var/log/file-d.meta"` //*
```

Comment:
Wrong comment, should be for `MetaDataDir`.  Also why don't do like in the original file config, just define full filename  (with path, etc) and some layout. https://github.com/ozontech/file.d/blob/master/plugin/output/file/file.go#L62 https://github.com/ozontech/file.d/blob/master/plugin/output/file/file.go#L69